### ISSUE TITLE
Export to gxftp fails if directory doesn't exist

### ISSUE THREAD

**Issue by mvdbeek** [galaxyproject/galaxy#11531]
I think users start out with the directory not existing, so a first export would fail with
<img width="857" alt="Screenshot 2021-03-03 at 10 49 39" src="https://user-images.githubusercontent.com/6804901/109787256-54dba800-7c0e-11eb-917f-2478d20d2592.png">

Not sure at what level to fix this

**Comment by jmchilton** [comment]
Yeah - I'm not sure if it is a Galaxy issue or an infrastructure issue even. The most conservative approach might be docs for setting that up with something like (https://github.com/galaxyproject/galaxy/pull/11310) the most aggressive approach might be to just have a new config option for creating FTP directories on use and defaulting it to ``true``. (With an option for user library directories also I suppose.)

@natefoo do you see any problems or shortcomings with just creating FTP directories by default as we need them?

**Comment by bgruening** [comment]
EU does actually have a cron job that creates those directories. I think there are other places that throw an exception when the folder is not there.

I just want to point out that having this flat hierarchy is leading to many thousand folders, so maybe we should also create subdirs based on the first letters.

**Comment by natefoo** [comment]
I actively clean up the individual user FTP directories using `tmpwatch`, ProFTPD creates them upon login if they're not there. It's perfectly reasonable to me to have Galaxy do this.
